packages/shared/src/components/modals/CompanionModal.tsx

```diff
@@ -41,110 +42,117 @@ export default function CompanionModal({
+        >
+          Read and activate widget
+        </Button>
 
-          <div className="flex flex-col flex-1 justify-between">
-            <div className="hidden tablet:flex flex-col items-center">
-              <div className="flex items-center pb-4">
-                <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
-                  <LineIcon
-                    size={IconSize.Medium}
-                    secondary
-                    className="text-theme-color-cabbage"
-                  />
-                </div>
+        <Button
+          className="mb-6 w-full btn-tertiaryFloat"
+          onClick={onReadArticleClick}
+          tag="a"
+          target="_blank"
+          href={url}
+          rel="noopener"
+        >
+          Read post
+        </Button>
 
-                <span>Get a TLDR on the original post</span>
+        <div className="flex flex-col flex-1 justify-between">
+          <div className="hidden tablet:flex flex-col items-center">
+            <div className="flex items-center pb-4">
+              <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
+                <UnreadIcon
+                  size={IconSize.Medium}
+                  secondary
+                  className="text-theme-color-cabbage"
+                />
               </div>
-              <div className="hidden tablet:flex items-center pb-4">
-                <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
-                  <CommentIcon
-                    size={IconSize.Medium}
-                    secondary
-                    className="text-theme-color-blueCheese"
-                  />
-                </div>
-                <span>Discuss from everywhere</span>
+
+              <span>Get a TLDR on the original post</span>
+            </div>
+            <div className="hidden tablet:flex items-center pb-4">
+              <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
+                <CommentIcon
+                  size={IconSize.Medium}
+                  secondary
+                  className="text-theme-color-blueCheese"
+                />
               </div>
-              <div className="hidden tablet:flex items-center pb-4">
-                <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
-                  <UpvoteIcon
-                    secondary
-                    size={IconSize.Medium}
-                    className="text-theme-color-avocado"
-                  />
-                </div>
-                <span>Bookmark with a single click</span>
+              <span>Discuss from everywhere</span>
+            </div>
+            <div className="hidden tablet:flex items-center pb-4">
+              <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
+                <BookmarkIcon
+                  secondary
+                  size={IconSize.Medium}
+                  className="text-theme-color-avocado"
```

Comment:
This should be orange color

Reply:
I can't find this color in the style guide -- double checking with design

Reply:
It's the bun color, just not sure if we added it as a theme color already.